nuxeo-core/nuxeo-core-storage-dbs/src/main/java/org/nuxeo/ecm/core/storage/dbs/DBSTransactionState.java

```diff
@@ -599,7 +617,15 @@ public void save() {
                 }
                 ChangeTokenUpdater changeTokenUpdater;
                 if (session.changeTokenEnabled) {
-                    changeTokenUpdater = new ChangeTokenUpdater(docState);
+                    // increment system change token
+                    Long base = (Long) docState.get(KEY_SYS_CHANGE_TOKEN);
```

Comment:
True but it's followed by a `put(KEY_SYS_CHANGE_TOKEN, ...)` so for symmetry I'll keep it that way.